.changeset/ten-dryers-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure-devops-backend': minor
+'@backstage/plugin-azure-devops': minor
+'@backstage/plugin-azure-devops-common': patch
+---
+
+Azure DevOps plugin is now integrated with permission framework for its core features, see the https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md for more details.
```

Comment:
This is great but I don't see any updates to this file around how to use the permissions?

Reply:
Documentation was added about the permissions and how to consume.